src/applications/mhv-secure-messaging/containers/EditContactList.jsx

```diff
@@ -232,36 +250,50 @@ const EditContactList = () => {
       {allTriageTeams?.length > 0 && (
         <>
           <form className="contactListForm">
-            {vistaFacilities.map(stationNumber => {
-              if (!blockedFacilities.includes(stationNumber)) {
-                const facilityName = getVamcSystemNameFromVhaId(
-                  ehrDataByVhaId,
-                  stationNumber,
-                );
+            <va-accordion bordered>
+              {vistaFacilities.map((stationNumber, index) => {
+                if (!blockedFacilities.includes(stationNumber)) {
+                  const facilityName = getVamcSystemNameFromVhaId(
+                    ehrDataByVhaId,
+                    stationNumber,
+                  );
 
-                return (
-                  <FacilityCheckboxGroup
-                    key={stationNumber}
-                    errorMessage={checkboxError}
-                    facilityName={facilityName}
-                    multipleFacilities={vistaFacilities?.length > 1}
-                    updatePreferredTeam={updatePreferredTeam}
-                    triageTeams={allTriageTeams
-                      .filter(
-                        team =>
-                          team.stationNumber === stationNumber &&
-                          team.blockedStatus === false,
-                      )
-                      .sort((a, b) => {
-                        const aName = a.suggestedNameDisplay || a.name;
-                        const bName = b.suggestedNameDisplay || b.name;
-                        return aName.localeCompare(bName);
-                      })}
-                  />
-                );
-              }
-              return null;
-            })}
+                  return (
+                    <va-accordion-item
+                      header={`${facilityName ||
+                        'VA Medical Center'} - ${stationNumber || ''}`}

```

Comment:
we need to return a healthcare system name here. there are references in our code where we use helper functions from `src/platform` like `getVamcSystemNameFromVhaId` that do that lookup from Drupal